src/arrayMethodJoin.js

```diff
@@ -5,8 +5,26 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```
You can just do like this